libs/eslint-config-typescript/package.json

```diff
@@ -11,9 +11,9 @@
 		"url": "git+https://github.com/guardian/csnx.git"
 	},
 	"license": "MIT",
-	"main": "index.js",
+	"main": ".eslintrc.js",
 	"dependencies": {
-		"@guardian/eslint-config": "^1.0.1",
+		"@guardian/eslint-config": "workspace:*",
```

Comment:
need to check how changesets resolves this in publishing. `pnpm` [handles it](https://pnpm.io/workspaces#publishing-workspace-packages) when publishing, i think changesets uses pnpm to publish if appropriate, so it _should_ work?

Reply:
it looks like changesets will run `pnpm publish` if your workspace uses pnpm https://github.com/changesets/changesets/blob/ab8f2b29d5cf921d97987a665051b67a7ec55ed3/packages/cli/src/commands/publish/npm-utils.ts#L37-L57

Reply:
i've added a changeset to test this

Reply:
didn't publish, hopefully https://github.com/guardian/csnx/pull/46 will resolve it

Reply:
it did! https://github.com/guardian/csnx/releases/tag/%40guardian%2Feslint-config-typescript%401.0.4